doc/rst/build.rst

```diff
@@ -26,31 +26,31 @@ https://github.com/hpc/mpifileutils/releases
 mpiFileUtils optionally depends on libarchive, version 3.5.1.
 If new enough, the system install of libarchive may be sufficient,
 though even newer versions may be incompatible with the required version.
-To be certain of compatibility, it is recommended that one install libarchive-3.5.1 with commands like the following
+To be certain of compatibility, it is recommended that one install libarchive-3.7.7 with commands like the following
 
 .. code-block:: Bash
 
    #!/bin/bash
    mkdir install
    installdir=`pwd`/install
 
-   wget https://github.com/libarchive/libarchive/releases/download/v3.5.1/libarchive-3.5.1.tar.gz
-   tar -zxf libarchive-3.5.1.tar.gz
-   cd libarchive-3.5.1
+   wget https://github.com/libarchive/libarchive/releases/download/v3.5.1/libarchive-3.7.7.tar.gz
```

Comment:
I got error 404: Not Found when I tried this wget command.

Reply:
Big mistake. I did not refresh the whole version everywhere. Will fix this right away. Thank you for the review.

Reply:
FYI if you do a "Squash and merge", it will squash commits for you. And though it says "merge", it just commits it to main - no "merge commit"